### PR TITLE
build: in cli build script, fix relative path to root

### PR DESCRIPTION
**Pull request by martinvonz**
Since the recent move of `builds.rs` to `cli/build.rs`, incremental builds re-calcuate the version number and thus result in some re-compilation and lots of re-linking.

# Checklist

If applicable:
- [ ] I have updated `CHANGELOG.md`
- [ ] I have updated the documentation (README.md, docs/, demos/)
- [ ] I have updated the config schema (src/config-schema.json)
- [ ] I have added tests to cover my changes

### CODE REVIEW

**Review by ilyagr — APPROVED** [review]
This LGTM, but I'm not 100% sure how `build.rs` and these directives are supposed to work. This might benefit from a review by somebody who understands cargo better, but perhaps Martin is that person.

**Comment by martinvonz** [comment]
> This LGTM, but I'm not 100% sure how `build.rs` and these directives are supposed to work. This might benefit from a review by somebody who understands cargo better, but perhaps Martin is that person.

I wouldn't say I know much about how `cargo` and `build.rs` work. I'm happy to wait for another review.

FYI, the documentation is here: https://doc.rust-lang.org/cargo/reference/build-scripts.html#rerun-if-changed

**Comment by ilyagr** [comment]
OK, now considering our Discord conversation, I checked pretty thoroughly that this works in practice. I think it can be merged.

**Pull request merged**
